ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksPage.tsx

```diff
@@ -94,7 +94,11 @@ function ProfileChecksPage() {
     };
 
     function handleProfilesToggleChange(selectedProfile: string) {
-        navigateWithScanConfigQuery(coverageProfileChecksPath, { profileName: selectedProfile });
+        navigateWithScanConfigQuery(
+            coverageProfileChecksPath,
+            { profileName: selectedProfile },
+            searchFilter
```

Comment:
We actually don't want to preserve filters when swapping profiles, but it's nice to know it's easy to do. Alan brought this up recently and pointed out that preserving a profile check filter in particular wouldn't make sense.